components/browser/session-storage/src/test/java/mozilla/components/browser/session/storage/serialize/BrowserStateWriterReaderTest.kt

```diff
@@ -175,6 +175,32 @@ class BrowserStateWriterReaderTest {
         assertEquals(333L, restoredTab.lastMediaAccessState.lastMediaAccess)
         assertTrue(restoredTab.lastMediaAccessState.mediaSessionActive)
     }
+
+    @Test
+    fun `Read and write tab with createdAt`() {
```

Comment:
`RestoringBrowsingSessionsTest` saves and restores some actual snapshots from devices. We could add some from Fenix with `created_at` integrated, to cover those too.

Reply:
Thx for the help @pocmo. Lemme know if it looks okay 😕